app/test_helpers.go

```diff
@@ -0,0 +1,34 @@
+package app
+
+import (
+	abci "github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/libs/log"
+	dbm "github.com/tendermint/tm-db"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/simapp"
+)
+
+// Setup initializes a new SimApp. A Nop logger is set in SimApp.
+func Setup(isCheckTx bool) *App {
```

Comment:
Is this used for anything other than tests?  Does it need access to private fields or functions?

Not sure it should be here.  If nothing else, call it `SetupTest()` and put it in a file that ends in `_test.go` so that it's only available during testing.